nilearn/_utils/tests/test_niimg_conversions.py

```diff
@@ -369,7 +369,7 @@ def test_check_niimg_wildcards_no_expand_wildcards(
 def test_iter_check_niimgs_error():
     no_file_matching = "No files matching path: %s"
 
-    for empty in ((), [], (i for i in ()), [i for i in ()]):
+    for empty in ((), [], (i for i in ()), []):
```

Comment:
```suggestion
    for empty in ((), [], (i for i in ())):
```
Do we need the last empty list now? The second element is exactly the same `[]`.

Reply:
yeah you are correct